pytradfri/gateway.py

```diff
@@ -150,6 +151,26 @@ def process_result(result):
         return Command('get', [ROOT_SMART_TASKS, task_id],
                        process_result=process_result)
 
+    def reboot(self):
+        """
+        Reboot gateway.
+
+        Returns a Command.
+        """
+        return Command('post',
+                       PATH_GATEWAY_INFO[0] + '/' + ATTR_GATEWAY_REBOOT)
```

Comment:
Wondering if we should refactor PATH_GATEWAY_INFO in const.py, and have reboot and wipe be constants with paths there instead? Doing `[0]` isn't very clean.